packages/core/src/dialects/ibmi/data-types.ts

```diff
@@ -8,16 +8,27 @@ export class UUID extends BaseTypes.UUID {
 }
 
 export class BOOLEAN extends BaseTypes.BOOLEAN {
+  protected supportsNativeBooleans() {
+    const databaseVersion
+      = Number.parseFloat(this._getDialect().sequelize.options.databaseVersion || this._getDialect().defaultVersion);
```

Comment:
Don't get the database version like this, but via `sequelize.getDatabaseVersion()`